flowmachine/flowmachine/core/cache.py

```diff
@@ -602,7 +604,7 @@ def get_compute_time(connection: "Connection", query_id: str) -> float:
     ----------
     connection : "Connection"
     query_id : str
-        md5 identifier of the query
+        query_id identifier of the query
```

Comment:
Maybe similar here? "query_id identifier" sounds less of a duplication than "query_id id", but maybe just use "identifier"?